src/main/java/com/atlassian/bitbucket/jenkins/internal/extensions/buildstatus/BitbucketPostBuildStatusListener.java

```diff
@@ -0,0 +1,44 @@
+package com.atlassian.bitbucket.jenkins.internal.extensions.buildstatus;
```

Comment:
The namespace here kind of assumes we'll have more than one extension. It may be more appropriate to move it to a different package